ibis/backends/clickhouse/tests/test_functions.py

```diff
@@ -532,3 +532,31 @@ def test_count_distinct_with_filter(alltypes):
     expected = alltypes.string_col.execute()
     expected = expected[expected.astype('int64') > 1].nunique()
     assert result == expected
+
+
+@pytest.mark.parametrize(
+    # FIXME: The `where` param needs `Nullable` column
+    #   but the all in testing data is not.
```

Comment:
Is this related to the xfail? You should be able to add a message to the xfail itself, and if for whatever reason that's not an option, don't make it a `FIXME`, which in general is for tasks that need to be done before it's merged.

In the xfail message, besides saying why the test is failing, you can add the issue number you created for it.